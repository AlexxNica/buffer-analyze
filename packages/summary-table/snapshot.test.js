// use story.js files as snapshots
import initStoryshots from '@storybook/addon-storyshots';
import shallow from 'react-test-renderer/shallow';

initStoryshots({
  suit: 'Snapshots',
  test({ story, context }) {
    const shallowRenderer = shallow.createRenderer();
    const result = shallowRenderer.render(story.render(context));
    expect(result).toMatchSnapshot();
  },
});
