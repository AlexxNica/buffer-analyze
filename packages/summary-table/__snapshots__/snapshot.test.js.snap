// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots SummaryTable should render a "no data" state 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <SummaryTable
      endDate={null}
      loading={false}
      metrics={Array []}
      profileService="twitter"
      startDate={null}
    />
  </div>
</span>
`;

exports[`Snapshots SummaryTable should render a loading state 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <SummaryTable
      endDate={null}
      loading={true}
      metrics={Array []}
      profileService="facebook"
      startDate={null}
    />
  </div>
</span>
`;

exports[`Snapshots SummaryTable should render the summary table 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <SummaryTable
      endDate={null}
      loading={false}
      metrics={
        Array [
          Object {
            "diff": 25,
            "label": "Tweets",
            "value": 150,
          },
          Object {
            "diff": -39,
            "label": "Retweets",
            "value": 901,
          },
          Object {
            "diff": -55,
            "label": "Clicks",
            "value": 1010,
          },
          Object {
            "diff": -26,
            "label": "Impressions",
            "value": 963400,
          },
          Object {
            "diff": 0,
            "label": "New Followers",
            "value": 0,
          },
          Object {
            "diff": -33,
            "label": "Engagements",
            "value": 28800,
          },
          Object {
            "diff": -28,
            "label": "Likes",
            "value": 2313,
          },
          Object {
            "diff": -9,
            "label": "Replies",
            "value": 658,
          },
        ]
      }
      profileService="twitter"
      startDate={null}
    />
  </div>
</span>
`;
